package.json

```diff
@@ -16,7 +16,7 @@
     "test": "npm run lint && npm run test-only",
     "test-only": "mocha -c --require test/setup.js --recursive ./test",
     "test-app": "./bin/nodejs-dashboard.js node test/app/index.js",
```

Comment:
Not part of this PR, but can you remove the prefix `./` in this command? `./` is unnecessary on mac/linux and breaks windows.